src/enums/ab-attr-flag.ts

```diff
@@ -327,4 +327,6 @@ export enum AbAttrFlag {
   REFLECT_STAT_STAGE_CHANGE,
   /** @see {@linkcode BypassParaSpeedReductionAbAttr} */
   BYPASS_PARA_SPEED_REDUCTION,
+  /** @see MockStatusEffectAbAttr */
```

Comment:
```suggestion
  /** @see {@linkcode MockStatusEffectAbAttr} */
```
Will need to update the tsdoc imports.